README.md

```diff
@@ -73,13 +73,13 @@ system
 vf.draw();
 ```
 
-[See it running here.](https://vexflow.github.io/vexflow-examples/static/step0)
+[See it running here.](https://vexflow.github.io/vexflow-examples/getting-started)
 
 [Learn more about EasyScore here.](https://github.com/0xfe/vexflow/wiki/Using-EasyScore)
 
 ## Native API
 
-If you need more control, you can use the low-level VexFlow API. Below, we render a stave using SVG. [See a running example of the low-level API here.](https://jsfiddle.net/5zgf03un/)
+If you need more control, you can use the low-level VexFlow API. Below, we render a stave using SVG. [See a running example of the low-level API here.](https://vexflow.github.io/vexflow-examples/getting-started)
```

Comment:
Line 82 and Line 76 refer to the same URL. Should they refer to a different URL (or at least a different `#section` since they are talking about different examples?

Reply:
There is also a typo on the vexflow-examples page. It says "It is also possible to use face fonts"

Reply:
But this is still possible, is not it? Perhaps we should reduce the README.md just to go directly into the vexflow-example sections and avoid the double maintenance of examples.

Reply:
@rvilarl What is a face font? :)

Reply:
font-face